scripts/configure-cluster/common/setup-operators.sh

```diff
@@ -1,46 +1,46 @@
 #!/bin/bash
 set -x
 
-install_etcd_operator(){
-  # Create subscription
-  oc create -f - <<EOF
+install_redis_operator(){
+  $1 create -f - <<EOF
   apiVersion: operators.coreos.com/v1alpha1
   kind: Subscription
   metadata:
-    name: etcd
-    namespace: openshift-operators
+    name: my-redis-operator
+    namespace: $2
   spec:
-    channel: clusterwide-alpha
-    installPlanApproval: Automatic
-    name: etcd
-    source: community-operators
-    sourceNamespace: openshift-marketplace
+    channel: stable
+    name: redis-operator
+    source: $3
+    sourceNamespace: $4
 EOF
 }
 
 install_service_binding_operator(){
-  oc create -f - <<EOF
+  $1 create -f - <<EOF
   apiVersion: operators.coreos.com/v1alpha1
   kind: Subscription
   metadata:
-    labels:
-      operators.coreos.com/rh-service-binding-operator.openshift-operators: ""
-    name: rh-service-binding-operator
-    namespace: openshift-operators
+    name: my-service-binding-operator
+    namespace: $2
   spec:
     channel: beta
-    installPlanApproval: Automatic
-    name: rh-service-binding-operator
-    source: redhat-operators
-    sourceNamespace: openshift-marketplace
+    name: $3
+    source: $4
+    sourceNamespace: $5
 EOF
 }
 
+if [ $KUBERNETES == "true" ]; then
+  # install redis-oprator
+  install_redis_operator kubectl operators operatorhubio-catalog olm
 
-# install etcd operator
+  # install service-binding-operator
+  install_service_binding_operator kubectl operators service-binding-operator operatorhubio-catalog olm
+else
+  # install redis-oprator
+  install_redis_operator oc openshift-operators community-operators openshift-marketplace
 
-install_etcd_operator
-
-# install service-binding-operator
-
-install_service_binding_operator
+  # install service-binding-operator
+  install_service_binding_operator oc openshift-operators rh-service-binding-operator redhat-operators openshift-marketplace
```

Comment:
SBO is available in the `community-operators` catalog source as well. We should use that since it ensures that catalog source is exactly same as the one used for minikube/k8s.

Reply:
> SBO is available in the `community-operators` catalog source as well. We should use that since it ensures that catalog source is exactly same as the one used for minikube/k8s.

Yes, I am using the same `community-operator` . Its just that name in the `spec` field with ocp is `rh-service-binding-operator` and minikube `service-binding-operator` . I was also confused when trying to deal with this subscription but then found this. Please let me know if still it doesn't make sense to you

Reply:
`redhat-operators` you've used in this line will be used as `source` (`$4`) in the `install_service_binding_operator` function. This `source` indicates the source Catalog Source for the subscription. What I'm requesting is to use `community-operators` catsrc in place of `redhat-operators`. 

So basically, I'm asking for:
```suggestion
  install_service_binding_operator oc openshift-operators rh-service-binding-operator community-operators openshift-marketplace 
```

Makes sense?

Reply:
I checked on multiple 4.7 clusters and it seems like the `community-operators` catalog source doesn't have SBO. It's only the `redhat-operators` one that has it. Earlier it was available via both the sources.

@prietyc123 ignore this change request.